docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -284,6 +482,32 @@ When a ConfigMap already being consumed in a volume is updated, projected keys a
 
 ## Understanding ConfigMaps and Pods
 
+ConfigMaps allow you to decouple configuration artifacts from image content to keep containerized applications portable.
```

Comment:
I think we could remove this **Understanding** section. It repeats ideas and exact text from earlier in the topic.